tests/e2e/device-plugin_test.go

```diff
@@ -162,12 +162,16 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 						"nvidia.com/gpu": "^[1-9]$",
 					}}
 				eventuallyNonControlPlaneNodes(ctx, f.ClientSet).Should(MatchCapacity(capacityChecker, nodes), "Node capacity does not match")
-			})
-			It("it should run GPU jobs", func(ctx context.Context) {
-				By("Creating a GPU job")
+
```

Comment:
@ArangoGutierrez just a note. There is an issue with the tests uninstalling the plugin between stanzas. This "fixes" them, but we should follow up at some point.